x-pack/test_serverless/api_integration/test_suites/security/cloud_security_posture/benchmark.ts

```diff
@@ -0,0 +1,223 @@
+        .post(`/api/fleet/agent_policies`)
+        .set('kbn-xsrf', 'xxxx')
+        .send({
+          name: 'Test policy',
+          namespace: 'default',
+        });
+
+      agentPolicyId = agentPolicyResponse.item.id;
+
+      const { body: agentPolicyResponse2 } = await supertest
+        .post(`/api/fleet/agent_policies`)
+        .set('kbn-xsrf', 'xxxx')
+        .send({
+          name: 'Test policy 2',
+          namespace: 'default',
+        });
+
+      agentPolicyId2 = agentPolicyResponse2.item.id;
+
+      const { body: agentPolicyResponse3 } = await supertest
+        .post(`/api/fleet/agent_policies`)
+        .set('kbn-xsrf', 'xxxx')
+        .send({
+          name: 'Test policy 3',
+          namespace: 'default',
+        });
+
+      agentPolicyId3 = agentPolicyResponse3.item.id;
+
+      const { body: agentPolicyResponse4 } = await supertest
+        .post(`/api/fleet/agent_policies`)
+        .set('kbn-xsrf', 'xxxx')
+        .send({
+          name: 'Test policy 4',
+          namespace: 'default',
+        });
+
+      agentPolicyId4 = agentPolicyResponse4.item.id;
+
+      await createPackagePolicy(
+        supertest,
+        agentPolicyId,
+        'cspm',
+        'cloudbeat/cis_aws',
+        'aws',
+        'cspm',
+        'CSPM-1'
+      );
+
+      await createPackagePolicy(
+        supertest,
+        agentPolicyId2,
+        'kspm',
+        'cloudbeat/cis_k8s',
+        'vanilla',
+        'kspm',
+        'KSPM-1'
+      );
+
+      await createPackagePolicy(
+        supertest,
+        agentPolicyId3,
+        'vuln_mgmt',
+        'cloudbeat/vuln_mgmt_aws',
+        'aws',
+        'vuln_mgmt',
+        'CNVM-1'
+      );
+
+      await createPackagePolicy(
+        supertest,
+        agentPolicyId4,
+        'kspm',
+        'cloudbeat/cis_k8s',
+        'vanilla',
+        'kspm',
+        'KSPM-2'
+      );
+    });
+
+    afterEach(async () => {
+      await kibanaServer.savedObjects.cleanStandardList();
+      await esArchiver.unload('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
+    });
+
+    it(`Should return non-empty array filled with Rules if user has CSP integrations`, async () => {
+      const { body: res }: { body: GetBenchmarkResponse } = await supertest
+        .get(`/internal/cloud_security_posture/benchmarks`)
+        .set(ELASTIC_HTTP_VERSION_HEADER, '1')
+        .set('x-elastic-internal-origin', 'xxx')
+        .set('kbn-xsrf', 'xxxx')
+        .expect(200);
+
+      expect(res.items.length).equal(3);
+      expect(res.total).equal(3);
+    });
+
+    it(`Should return array size 2 when we set per page to be only 2 (total element is still 3)`, async () => {
+      const { body: res }: { body: GetBenchmarkResponse } = await supertest
+        .get(`/internal/cloud_security_posture/benchmarks?per_page=2`)
+        .set(ELASTIC_HTTP_VERSION_HEADER, '1')
+        .set('x-elastic-internal-origin', 'xxx')
```

Comment:
adding `x-elastic-internal-origin` is the only thing that is different for our tests (the ones that worked out of the box ) between stateful and serverless